lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.pop(2)'
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = '[12, 34, 99]'
 
```

Comment:
Close!! It would actually be `[12, 34, nil, nil, 99]` since arr[2] and arr[3] don't exist and we are setting arr[4], those values become `nil`.